spisea/imf/multiplicity.py

```diff
@@ -194,3 +196,120 @@ def random_companion_count(self, x, CSF, MF):
         n_comp = 1 + np.random.poisson((CSF / MF) - 1)
 
         return n_comp
+    
+class MultiplicityResolvedDK(MultiplicityUnresolved):
+    """
+    Sub-class of MultiplicityUnresolved that adds semimajor axis and eccentricity information 
+    for multiple objects from distributions described in Duchene and Krauss 2013
+    
+    Parameters
+    --------------
+    a_amp - float, optional
```

Comment:
Standard commenting should be:

a_amp : float, optional
    some comment here...

Change so auto-docs work.